project2/get_ingredients.py

```diff
@@ -0,0 +1,112 @@
+# input "1 cup white sugar"
+# output
+    # ingredient: sugar
+    # quantity: 1
+    # measurement: cup
+    # descriptior: white
+    # preparation: ''
+
+# input "1 cup butter, softened"
+# output
+    # ingredient: butter
+    # quantity: 1
+    # measurement: cup
+    # descriptior: ''
+    # preparation: 'soften'
+
+import re
+import nltk
+from nltk.corpus import stopwords
+from collections import Counter
+
+def clean_ingredients(ing):
+    # basic cleaning
+        # remove all punctiation
+        # keep / : 1/5 pounds
+        # keep . : 1.5 pounds
+    pattern = '[,;!$%^&*()]'
+    ing = ' '.join(re.sub(pattern," ",ing).split())
+    return ing
+
+
+def normalize_ingredients(ing):
+    ing = clean_ingredients(ing)
+    # convert all words to lower case
```

Comment:
I don't think we need these comments because they are restating the obvious. it is pretty clear that `lower` will make a string lowercase and `word_tokenize` will tokenize a string.